kernel/chips/pulp_cluster/link.ld

```diff
@@ -221,12 +221,16 @@ SECTIONS
     . = ALIGN(4);
   } > L2
 
+  __l2_data_end = .;
+
   .l1cluster_g : {
     . = ALIGN(4);
+    __heap_sram = .;
     *(.heapsram)
     *(.heapsram.*)
     *(.l1cluster_g)
     *(.l1cluster_g.*)
+    __data_l1 = .;
```

Comment:
`.data_l1` was not working?

Reply:
No, and this is also the reason why we are calling `_data_l1` [here](https://github.com/pulp-platform/pulp-runtime/pull/38/files/968f04e35a19cfd224f07ee691321b55759f1ebd#diff-ad44de61d3fa74722c3f9b2a7012a7555dd71c57af034aeab4fc6bc9e9d08745L42-R42). The reason is not totally clear to me (I already asked clarifications to @gtagliavini), but if we do not create a global symbol to refer to in the code, using `.data_l1` results in the linker statically allocating that section in the 0x1000_XXXX memory region, which does not exist if we shift the cluster index.

Reply:
We need to do the same in the [regression test](https://github.com/pulp-platform/regression_tests/commit/df16a13c672ac360112b5b9e80fb46b72d266bf7#diff-cd25130498edfefd1eecd27deac2f96e963730626e0a6e83b774e9786f4b962dR32)

Reply:
so the problem is that we cannot declare the l1 section as static anymore right?

Reply:
Exactly. For sure this does not work if a single cluster has a non-zero index, but I think it would break also if we have multiple clusters.

Reply:
@bluewww any input on how to handle this?